account/accountservice_test.go

```diff
@@ -155,7 +156,7 @@ func TestVerfiyFail(t *testing.T) {
 	assert.NotNil(t, tx.Body.Sign, "failed to sign")
 
 	//edit tx after sign
-	tx.Body.Amount = 0xff
+	tx.Body.Amount = new(big.Int).SetUint64(0xff).Bytes()
```

Comment:
Isn't `big.NewInt(0xff).Bytes()` the same as `new(big.Int).SetUint64(0xff).Bytes()`?

Reply:
right 👍 I will fix :)